cypress/integration/manualTests/monitor.ts

```diff
@@ -50,3 +52,27 @@ describe('Monitor homepage links', () => {
     });
   });
 });
+
+describe('Test launch button icons', () => {
+  it('find a WDL workflow', () => {
+    cy.visit('/search');
+    cy.wait(10000); // wait for the tabs to become clickable
```

Comment:
should wait for something specific instead.  like the tab has content (which may imply that it is clickable) or something else.